sprokit/processes/adapters/register_processes.cxx

```diff
@@ -46,34 +46,19 @@ extern "C"
 KWIVER_PROCESSES_ADAPTER_EXPORT
 void
 register_factories( kwiver::vital::plugin_loader& vpm )
-
 {
-  static auto const module_name = kwiver::vital::plugin_manager::module_t( "kwiver_processes_adapters" );
+  using namespace sprokit;
```

Comment:
This doesn't seem used?

Reply:
process registrar is in sprokit name space